site/docs/v1/tech/installation-guide/monitor.adoc

```diff
@@ -79,3 +79,18 @@ You can also enable JMX metrics as outlined in the link:/docs/v1/tech/troublesho
 ==== Prometheus Endpoint
 
 The default system metrics are also available in a Prometheus compatible form. link:/docs/v1/tech/tutorials/prometheus[This tutorial explains how to set up Prometheus to scrape the FusionAuth metrics].
+
+=== Load Testing
+
+You can load test FusionAuth to determine if its performance will meet your calculated needs. The FusionAuth team tests the performance of FusionAuth regularly, but every situation is different and running your own load tests on your instance provides useful information.
+
+You may use your own load testing tool. Any tool which can make HTTP requests, such as gatling or jmeter, will work.
+
+There is an https://github.com/FusionAuth/fusionauth-load-tests/[open source project] which provides some useful scripts for load testing. This is a work in progress, so use at your own risk. Scenarios currently supported include:
```

Comment:
Instead of saying "work in progress" could we say, "this is a purpose built library and used for internal purposes, use at your own risk". 

While, it is always in progress, it is fully functional, at least for our specific purposes.